Convert dashboard filter checkboxes to toggle buttons

carpeliam opened the pull request.

carpeliam committed:
change dashboard measure filter behavior from checkbox to button

carpeliam committed:
checkbox buttons should be btn-blocks

carpeliam committed:
remove panel padding

carpeliam committed:
fix dashboard measure filter

noranda commented:
The functionality works great. There does need to be a equal padding for each button though. It looks as if there is no left padding.

carpeliam committed:
restore padding to buttons removed by btn-block

Future versions of bootstrap don't remove padding from btn-block, but we can't use those versions.

carpeliam commented:
@noranda that should be addressed in 034063f.

noranda commented:
Looks good!

carpeliam committed:
wrap links in actual block level element to support jQuery show/hide
